### PR TITLE
webgpu: Fix leakyRelu error for conv2d vec4

### PR DESCRIPTION
**Pull request by qjia7**
To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/6292)

### CODE REVIEW

File: tfjs-backend-webgpu/src/kernels/Conv2D_impl.ts [hunk 1]
```diff
@@ -270,7 +270,7 @@ export function conv2DImpl({
   const useVec4 =
       (convInfo.inChannels % 4 === 0 ||
        (convInfo.inChannels === 3 && convInfo.padInfo.type === 'VALID')) &&
-      convInfo.outChannels % 4 === 0 && convInfo.outChannels >= 32;
```
Comment (qjia7) [inline tfjs-backend-webgpu/src/kernels/Conv2D_impl.ts:273]: Per our test, Conv2DMMVec4Program is always faster than Conv2DMMProgram based on current configuration. So it's not necessary to add this limitation. And without this limitation, the current test `fused conv2d webgpu {"WEBGPU_CPU_FORWARD":false} leakyrelu bias stride 2 x=[1,8,8,16] f=[3,3,16,1] s=[2,2] d=8 p=same` can catch this leakyRelu bug.

**Comment by qjia7** [comment]
@gyagp @axinging @xhcao @haoyunfeix PTAL, thanks.

**Comment by axinging** [comment]
LGTM,  thanks!

**Review by gyagp — APPROVED** [review]
LGTM

**Pull request merged**